concepticondata/conceptlists.tsv

```diff
@@ -371,3 +371,5 @@ Vulic-2020-2244	Vulić, Ivan and Baker, Simon and Ponti, Edoardo Maria and Petti
 Huttenlocher-1979-225	Huttenlocher, Janellen and Lui, Felicia	1979	acquisition	225		English	English		Huttenlocher1979	Huttenlocher1979	This list contains stimuli from three experiments on the semantic organization of nouns and verbs in early language acquisition. In the list, we have marked the semantic groups to which individual items were assigned in the three experiments. These categories were added by us, as they were not provided in the original data, but rather tacitly assumed.	141-162	
 Carvalho-2021-206	de Carvalho, Fernando O.	2021		206	basic, areal	English	Iñapari, Apurinã, Yine		Carvalho2021		The given list consists of 206 cognate sets from the Iñapari, Apurinã, and Yine varieties. The data were employed in a comparative reconstruction of the phonology of their shared ancestor, the Proto-Purus language. Since the data were gathered from multiple dictionaries and grammars on the target languages, some concepts marked with (sp.) appear to refer to different species of the same animal.	96-106	
 Thwaites-2020-98	Thwaites, Peter	2020		98	specific	English	English		Thwaites2020		The given 98-item list consists of 49 transitive and 49 intransitive verbs that were employed in a word association task in order to investigate the impact of transitivity of English verbs on response patterns among native English speakers. The intransitive verbs were derived from an online list of verbs based on the search query “English intransitive verbs”. The transitive verbs were selected from the [SUBTLEX-UK database](:bib:vanHeuven2014), and the Longman Dictionary of Contemporary English was used to eliminate the verbs with intransitive entries.	481	
+Scheible-2014-1755a	Scheible, Silke and Schulte im Walde, Sabine	2014	a	1755	ratings	German	German		Scheible2014		The database includes semantically related word pairs for German. The ratings on similarity are based on human judgement experiments and comprise (i) a representative selection of target lexical units balanced for semantic category, polysemy, and corpus frequency, (ii) a set of humangenerated semantically related word pairs based on the target units, and (iii) a subset of the generated word pairs rated for their relation strength, including positive and negative relation evidence.		Database of Paradigmatic Semantic Relation Pairs for German
```

Comment:
Must be a typo in "humangenerated" that is missing a hyphen.
I'm also not sure that the list suffixes are needed here since the lists technically differ in their IDs.

Reply:
I followed the procedure in a recent PR of the Bowern hunter-gatherer list, see #1047.